packages/bindings/src/gov.rs

```diff
@@ -86,6 +85,48 @@ pub enum GovProposal {
         /// all code ideas that should be removed from cache to free space
         code_ids: Vec<u64>,
     },
+    // ClientUpdateProposal is a governance proposal. If it passes, the substitute
+    // client's latest consensus state is copied over to the subject client. The proposal
+    // handler may fail if the subject and the substitute do not match in client and
+    // chain parameters (with exception to latest height, frozen height, and chain-id).
+    ClientUpdate {
```

Comment:
Isn't this meant to replace `IbcClientUpdate`, above?

Reply:
`title` and `description` seems to be in every message and in rust implementation they are just extracted directly to `ExecuteGovProposal` (https://github.com/confio/poe-contracts/blob/ibc-client-update-proposal-fix/packages/bindings/src/msg.rs#L36-L37). It is not 100% clear to me.